mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/presenter/ProfileViewModel.kt

```diff
@@ -0,0 +1,81 @@
+package org.mifos.mobilewallet.mifospay.home.presenter
+
+import android.graphics.Bitmap
+import android.graphics.BitmapFactory
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import dagger.hilt.android.lifecycle.HiltViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.launch
+import okhttp3.ResponseBody
+import org.mifos.mobilewallet.core.base.UseCase
+import org.mifos.mobilewallet.core.base.UseCaseHandler
+import org.mifos.mobilewallet.core.domain.usecase.client.FetchClientImage
+import org.mifos.mobilewallet.datastore.PreferencesHelper
+import org.mifos.mobilewallet.mifospay.data.local.LocalRepository
+import org.mifos.mobilewallet.mifospay.utils.DebugUtil
+import javax.inject.Inject
+
+@HiltViewModel
+class ProfileViewModel @Inject constructor(
+    private val mUsecaseHandler: UseCaseHandler,
+    private val fetchClientImageUseCase: FetchClientImage,
+    private val localRepository: LocalRepository,
+    private val mPreferencesHelper: PreferencesHelper
+) : ViewModel() {
+
+    private val _bitmapImage = MutableStateFlow<Bitmap?>(null)
```

Comment:
Create ProfileUiState sealed class and move these fields into that.

Reply:
Example: https://github.com/android/nowinandroid/blob/main/feature/foryou/src/main/kotlin/com/google/samples/apps/nowinandroid/feature/foryou/OnboardingUiState.kt